minetest/minetest
# AppImageBuilder.yml: add script section and update to Jammy

Add compact, short information about your PR for easier understanding:

- This PR fixes some things into the appimage-builder recipe for building a Minetest AppImage
- How does the PR work?
- AppImage option can be added to Minetests' download page under Linux and possibly FreeBSD (may be later though)
- This solves the problem for GNU/Linux for a universial package that is reliable, up to-date, and doesn't require any other software to work (may work on FreeBSD and others but is untested)

## To do

This PR is Ready for Review.

- [X] Add general fixes to AppImageBuilder recipe

## How to test

First this is built for linux. I recommend Ubuntu 22.04 (Jammy) for building as the supporting libraries in the AppImage are pulled from Ubuntu Jammy.
Make sure you can build minetest and that (appimage-builder)[https://appimage-builder.readthedocs.io] can be ran.

Then run:
`appimage-builder` inside minetest's build directory. Then there should be a `Minetest-5.6.1-x86_64.AppImage` and `Minetest-5.6.1-x86_64.AppImage.zsync` file.
